geoips/commandline/log_setup.py

```diff
@@ -34,7 +34,12 @@ def log_with_emphasis(print_func, *messages):
         The messages to be logged with emphasis
     """
     wrapped_messages = []
-    messages = filter(lambda s: len(s) > 0, messages)
+    try:
+        messages = filter(lambda s: len(s) > 0, messages)
+    except TypeError:
+        raise TypeError(
+            f"1+ words sent to log_with_emphasis doesn't have a length attr. {messages}"
```

Comment:
Why add this?

Reply:
I was getting a TypeError in recenter_tc when I supplied an ``area_def`` in messages to ``log_with_emphasis``. All in all, this is very unlikely to occur, but will raise a more appropriate error if anything in ``*messages`` doesn't have a ``len`` attribute.

Reply:
Ah - is this when you pass something that isn't a string? Perhaps the filter should use 
```python
lambda s: len(s)>0 or len(str(s))>0
```
What do you think?

Reply:
Sure, that's probably a better solution. I'll make that update now.

Reply:
Updated!